html/comparison.html.haml

```diff
@@ -357,7 +357,7 @@
                 %span.summary-part{"ng-show" => "diff.groups.only_in2"}
                   {{diff.meta.description_b}}: {{diff.groups.only_in2.length}} groups
                 %span.summary-part{"ng-show" => "diff.groups.common"}
-                  %a{:class=>"show-common-elements", href: "#groups_both"}
+                  %a{:class=>"show-common-elements", href: "#groups_both"}<
```

Comment:
Avoid defining `class` in attributes hash for static class names<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace